includes/Dashboard/Templates/Settings.php

```diff
@@ -58,8 +58,10 @@ public function render_settings_header() {
         if ( isset( $wp->query_vars['settings'] ) && $wp->query_vars['settings'] == 'store' ) {
             $heading          = __( 'Settings', 'dokan-lite' );
             $is_store_setting = true;
-        } elseif ( isset( $wp->query_vars['settings'] ) && $wp->query_vars['settings'] == 'payment' ) {
-            $heading = __( 'Payment Settings', 'dokan-lite' );
+        } elseif ( isset( $wp->query_vars['settings'] ) && 'payment' === substr( $wp->query_vars['settings'], 0, 7 ) ) {
```

Comment:
the value inside `query_vars['settings']` can not only be 'payment', it can be 'payment/manage-paypal' or 'payment/manage-paypal/edit' or other payment method too. so I am using the prefix of length 7 to compare the value.